airbyte-integrations/connectors/source-twilio/source_twilio/streams.py

```diff
@@ -90,45 +100,53 @@ def custom_transform_function(original_value: Any, field_schema: Mapping[str, An
                 # is no need in transforming anything.
                 pass
         return original_value
+        
 
 
-class IncrementalTwilioStream(TwilioStream, ABC):
+class IncrementalTwilioStream(TwilioStream, IncrementalMixin):
     cursor_field = "date_updated"
     time_filter_template = "%Y-%m-%dT%H:%M:%SZ"
-
-    def __init__(self, start_date: str = None, **kwargs):
+    
+    def __init__(self, start_date: str = None, lookback: int = 0, **kwargs):
         super().__init__(**kwargs)
-        self._start_date = start_date
+        self._start_date = start_date  
+        self._lookback = lookback
```

Comment:
```suggestion
        self._lookback_window = lookback_window
```
It's more explicit to name in `lookback_window` for consistency with other connectors sharing the same kind of logic.